pages/governmentresumeguide.md

```diff
@@ -0,0 +1,75 @@
+#Government-style Resume Guide
+
+**Full legal name (First Middle Last)** <br>
```

Comment:
Update: 
- let's remove "full legal name (first middle last)"
- scratch adding the "preferred name"
- Replace it with "name"